<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceHelper</class>
 <widget class="QMainWindow" name="InterfaceHelper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
    border: 3px solid transparent;
	border-color: #373B44;
    padding: 4px 10px;
    font-size: 12px;
    color: #373B44;
    background-color: transparent;
    background-image: conical-gradient(from 90deg, transparent 90deg, #373B44 90deg);
    background-position: center;
    background-repeat: no-repeat;
    background-origin: content;
    outline: none;
}

QPushButton:hover {
    outline: none;
	background-color: #bbbfba;
}

QPushButton:pressed {
    background-color: #737573;
    color: white;
}

/* QGroupBox */
QGroupBox {
    border: 1px solid #666;
    border-radius: 6px;
    margin-top: 6px;
    padding: 6px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 6px;
    color: #1d1f1d;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,1,0" columnstretch="0,1,1,0">
    <item row="0" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1" colspan="2">
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,2" columnstretch="3,2" rowminimumheight="0,1" columnminimumwidth="0,0">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Загальна інформація</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1,1,1,0">
           <item>
            <widget class="QPushButton" name="pushButton_about">
             <property name="text">
              <string>Про програму</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_works">
             <property name="text">
              <string>Принцип роботи</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_abilities">
             <property name="text">
              <string>Можливості програми</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_how">
             <property name="text">
              <string>Як взаємодіяти</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Можливі параметри</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPushButton" name="pushButton_VertSync">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Вертикальна синхронізація</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_AntMode">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Рівень згладжування</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_AnisFiltering">
             <property name="text">
              <string>Анізотропна фільтрація</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_TextFiltQuality">
             <property name="text">
              <string>Якість фільтрації текстур</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_AmbOcculusion">
             <property name="text">
              <string>Амбієнтне затемнення</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_PowManagMode">
             <property name="text">
              <string>Енергоспоживання</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_TripBuffering">
             <property name="text">
              <string>Потрійний буфер</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_ThrOpti">
             <property name="text">
              <string>Багатопоточна оптимізація</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_CUDA">
             <property name="text">
              <string>CUDA - ГПУ</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Опис</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="label_main">
               <property name="text">
                <string>TextLabel</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>758</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
